deploy/charts/templates/deployment.yaml

```diff
@@ -30,7 +30,7 @@ spec:
       - name: {{ .Chart.Name }}
         image: "{{ .Values.image.repository }}:{{ .Values.image.tag }}" # In Jenkins, need to call helm with --set image.tag=master-XY
         command: ["tini"]
-        args: ["--", "npm", "start", "--", "--port={{ .Values.service.port }}", "--config={{ .Values.deployenv.environment }}"]
+        args: ["--", "node", "server/index.js", "--port={{ .Values.service.port }}", "--config={{ .Values.deployenv.environment }}"]
```

Comment:
NODE_ENV is now provided globally and we can start the app directly. This removes the intermediary `npm` process and allows Terminus to exit the node process directly. This prevents a lingering `npm` process which would fail when it lost track of the app it "remembered" starting.

Reply:
I wonder if this is the same issue that causes a failed 'npm run test:e2e' to leave ghost npm processes as well.

Reply:
This seemed to be a key to managing node app shutdown from various resources I saw... but it also seemed nontrivial to implement, so 👍.